picking_good_slos.md

```diff
@@ -0,0 +1,57 @@
+# How to pick a good SLO
+
+A Service Level Objective (SLO) is a measurable, specific goal for a given service's reliability level of service over time, based on one or more SLIs.
+To learn more about SLIs, check out our article on how to [pick good SLIs](./picking_good_slis.md).
+
+An SLO is used to quantify customer experience.
+Furthermore, SLOs enable us to avoid false positives in alerting and help us focus on symptom-based alerting.
+SLOs can vary from organization to organization. Consequently, there is no single formula that we could use to define the appropriate reliability target. 
+In this article we will share some tips for picking a good SLO.
+
+So, what should we consider when picking the SLO for a service?
+
```

Comment:
How would you feel about adding a short paragraph here along the lines of:

As stated previously, the SLO is based on good SLIs, and these should reflect the aspects of the service of most importance to the customer. All the considerations following are nuances of this underlying theme.